tree/30_generic_methods/permissions_recurse.cf

```diff
@@ -54,7 +54,7 @@ bundle agent permissions_recurse(path, mode, owner, group)
       "reenable_reporting_${class_prefix}"
                                    usebundle => enable_reporting,
                                   ifvarclass => "should_report";
-      "new result classes"         usebundle => _classes_copy("${inner_class}", "${class_prefix}");
+      "new result classes"         usebundle => _classes_copy("${inner_class_prefix}", "${class_prefix}");
```

Comment:
this should not be part of this PR

Reply:
definitely, I'll open a ticket (I just saw this while copy/pasting the inner class code)